FNS-Part-1.md

```diff
@@ -992,7 +992,7 @@ In Ethereum, we are used to ERC-20 token balances being stored as part of the gl
 
 Similar to how the `NonFungibleToken` standard defines `Collection` - where each user has a `Collection` of their own stored in their storage, and that collection contains the NFTs they own from that smart contract, the `FungibleToken` has `Vault`s.
 
-A `Vault` is nothing more than a collection of tokens. You could define a `Vault` to be empty, to have some portion of the total tokens you own of that type, or to have all the tokens you own of that type. It is completely upto the user.
+A `Vault` is nothing more than a collection of tokens. You could define a `Vault` to be empty, to have some portion of the total tokens you own of that type, or to have all the tokens you own of that type. It is completely up to the user.
```

Comment:
Wow. That's right.  [Reference](https://www.grammar-monster.com/easily_confused/into_onto_up_to.htm#:~:text=Finally%2C%20an%20easy%20rule%3A%20%22,Upto%22%20is%20always%20wrong.)

I did not know that